.github/workflows/build-firmware.yaml

```diff
@@ -5,7 +5,7 @@ on: [push, pull_request, workflow_dispatch]
 jobs:
   build-firmware:
```

Comment:
Does `build-primary-bundle` job receive special treatment, or should it also be updated?